lib/ansible/modules/service_facts.py

```diff
@@ -9,11 +9,7 @@
 __metaclass__ = type
 
 
-ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['preview'],
-                    'supported_by': 'core'}
-
-DOCUMENTATION = r'''
+DOCUMENTATION = '''
```

Comment:
Why remove the `r` here?

Reply:
not on purpose .... but i've rebased 30x sooo, probably merged from somewhere

Reply:
```suggestion
DOCUMENTATION = r'''
```